webpack.config.js

```diff
@@ -64,6 +64,7 @@ const nodeModulesToTranspile = [
 	// general form is <package-name>/.
 	// The trailing slash makes sure we're not matching these as prefixes
 	// In some cases we do want prefix style matching (lodash. for lodash.assign)
+	'@automattic/calypso-polyfills',
```

Comment:
There needs to be a trailing slash here. If there isn't one, the code would also match a package name like `@automattic/calypso-polyfills-blah-blah` 🙂

Reply:
Oops, I missed that, sorry!